src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests.cs

```diff
@@ -651,7 +651,8 @@ class Program
             Assert.True(attributeData.HasErrors);
             Assert.Equal("AAttribute..ctor(params System.Int32[] args)", attributeData.AttributeConstructor.ToTestDisplayString());
             Assert.Equal(1, attributeData.AttributeConstructor.ParameterCount);
-            Assert.Equal(new object[] { 1, 2, 3 }, attributeData.ConstructorArguments.Select(arg => arg.Value));
+            Assert.Equal(TypedConstantKind.Array, attributeData.ConstructorArguments.Single().Kind);
```

Comment:
>attributeData.ConstructorArguments.Single() [](http://example.com/codeflow?start=50&length=43)

Is this a meaningful change to the public API behavior for `AttributeData.ConstructorArguments`?

Reply:
> Is this a meaningful change to the public API behavior for `AttributeData.ConstructorArguments`?

Yes. In some error conditions we are actually manage to create an array. The behavior is close to the success case now, see the other test I modified in this file